specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-10-01-preview/mfe.json

```diff
@@ -9,6 +9,90 @@
     "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/availableQuota/default": {
```

Comment:
>"/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/availableQuota [](http://example.com/codeflow?start=4&length=91)

Define a LIST for "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/availableQuota 


to ensure that azure policy works for your resource